Cql/CoreTests/Input/ValueSets/intensional-value-set.json

```diff
@@ -7,10 +7,14 @@
     "include": [
       {
         "valueSet": [
-          "https://www.ncqa.org/fhir/valueset/2.16.840.1.113883.3.464.1004.1009",
```

Comment:
This is wrong, it means select all codes that occur in BOTH valuesets, which is an empty collection. Instead, it should have been two separate `valueSet` statements, which is then handled as an OR.